core/templates/dev/head/profile/Preferences.js

```diff
@@ -35,6 +36,46 @@ oppia.controller('Preferences', ['$scope', '$http', '$rootScope', '$modal', '$ti
     _saveDataItem('user_bio', userBio);
   };
 
+  $scope.changedAtLeastOnce = false;
+  $scope.subjectInterestsWarningText = null;
+  $scope.TAG_REGEX = '^[a-z ]+$';
+
+  $scope.updateSubjectInterestsWarning = function(subjectInterests) {
+    var alpha = /^[A-Za-z]+$/;
+
+    if (subjectInterests instanceof Array) {
+      for (var i=0; i<subjectInterests.length; i++) {
+        if (typeof subjectInterests[i] === 'string') {
+          console.log(subjectInterests[i]);
+          if (alpha.test(subjectInterests[i])) {
+            if ($scope.subjectInterestsWarningText == null) {
```

Comment:
If there's nothing wrong you can keep the warning text as null. No changes are needed.